<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_ModifierProfil</class>
 <widget class="QDialog" name="f_ModifierProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>170</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/Images/App/icons_simple.png</normaloff>:/Images/App/icons_simple.png</iconset>
  </property>
  <widget class="QLabel" name="Lb_AncienMdP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>110</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Ancien mot de passe :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_AncienMdP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="BtBx_Valider">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_ConfimMdP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmer mot de passe :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_ConfirmMdP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_NvMdP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau mot de passe :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NvMdP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
